--- /configure.ac	2021-11-15 09:43:00.000000000 -0800
+++ /configure.ac	2022-04-25 13:33:17.189246017 -0700
@@ -403,6 +403,9 @@
 	*-*-vxworks*)
 	    ac_sys_system=VxWorks
 	    ;;
+	*-*-darwin*)
+	    ac_sys_system=Darwin  
+	    ;;  
 	*)
 		# for now, limit cross builds to known configurations
 		MACHDEP="unknown"
@@ -452,6 +455,9 @@
 	*-*-vxworks*)
 		_host_cpu=$host_cpu
 		;;
+	*-*-darwin*)
+		_host_cpu=$host_cpu
+		;;
 	*)
 		# for now, limit cross builds to known configurations
 		MACHDEP="unknown"
